### DISCUSSION TITLE
Approved but not pack yet , applied on december 2 ,

### DISCUSSION THREAD

**Discussion by kekii88** [community/community#146211] [GitHub Education]
### Hi everyone,

hello everyone i applied for pack on december 2 but its still not here yet
![2024-12-04 10_40_22-Get your GitHub benefits - GitHub Education — Firefox Developer Edition](https://github.com/user-attachments/assets/ebe3e0f5-b637-4888-95d1-641227b3e0bf)
is this my issue or everyones

**Comment by quoc10973** [comment]
they told that your benefit will available soon so you need to keep waiting. Here my aprroved and its said "are now available"
![image](https://github.com/user-attachments/assets/76ff3dc5-1503-4742-9c66-1d5e2f10bf26)

**Reply by kekii88** [reply]
how long it took for you?

**Reply by quoc10973** [reply]
its about 4 days, be patient !!

**Comment by davevad93** [comment]
Hi @kekii88 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kekii88 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
